core/jazz_metrics/components/utils.js

```diff
@@ -240,26 +240,55 @@ function updateCloudfrontAsset(newAssetObj, relativeId) {
   return newAssetObj;
 }
 
-function getCloudWatch() {
-  var cloudwatch = new AWS.CloudWatch({
-    apiVersion: '2010-08-01'
-  });
+function getCloudWatch(tempcreds) {
+  tempcreds.apiVersion = '2010-08-01';
+  var cloudwatch = new AWS.CloudWatch({tempcreds});
   return cloudwatch;
 }
 
-function getCloudfrontCloudWatch() {
-  var cloudwatch = new AWS.CloudWatch({
-    apiVersion: '2010-08-01',
-    region: global_config.CF_REGION
-  });
+function getCloudfrontCloudWatch(tempcreds , serviceMetaData) {
+  tempcreds.apiVersion = '2010-08-01';
+  tempcreds.region = serviceMetaData.services.region
+  var cloudwatch = new AWS.CloudWatch({tempcreds});
   return cloudwatch;
 }
 
+function AssumeRole(accountID) {
+  logger.info("Inside Assume Role")
+  return new Promise((resolve, reject) => {
+    const sts = new AWS.STS({ region: process.env.REGION });
+    const params = {
+      RoleArn: 'arn:aws:iam::'+accountID+':role/'+global_config.STACK_PREFIX+'_platform_services',
```

Comment:
RoleArn: `arn:aws:iam::${accountID}:role/${global_config.STACK_PREFIX}_platform_services`,

Reply:
This string has been removed on the newer commits.